CRM/Contact/BAO/Contact.php

```diff
@@ -3547,9 +3547,9 @@ public static function isFieldHasLocationType($fieldTitle) {
    * @return array|false
    */
   public static function getEntityRefCreateLinks($appendProfiles = []) {
-    // You'd think that "create contacts" would be the permission to check,
+    // You'd think that "add contacts" would be a sufficient permission to check,
     // But new contact popups are profile forms and those use their own permissions.
-    if (!CRM_Core_Permission::check([['profile create', 'profile listings and forms']])) {
+    if (!CRM_Core_Permission::check([['profile create', 'profile listings and forms, 'add contacts']])) {
```

Comment:
need a `'` after forms and before the `,`

Reply:
Sorry, not sure how to edit properly so https://github.com/civicrm/civicrm-core/pull/24005/commits/da650ece08d18d7f120b5744ec6f43fc59dc4fbf